libs/domains/organizations/feature/src/lib/helm-repository-create-edit-modal/helm-repository-create-edit-modal.tsx

```diff
@@ -142,6 +142,7 @@ export function HelmRepositoryCreateEditModal({
                 onChange={(value) => {
                   methods.resetField('config')
                   field.onChange(value)
+                  methods.clearErrors('url')
```

Comment:
it should be better to reset the field no ?

When I switch from `OCI` kind to `HTTPS`, I don't have error on my `Repository url` field, I need to re-update this to see the error

Reply:
Fixed https://github.com/Qovery/console/pull/1249/commits/e69a2dc2659e18c8781bb1ff44ae15f6d92fa0ce